### PR TITLE
Sets 1$ monitoring reward hardcoded for now

### PR DESCRIPTION
**Pull request by Dominik1999**
Closes: #4684

## Description

Please, describe what this PR does in detail:
- If it's a bug fix, detail the root cause of the bug and how this PR fixes it.
- If it's a new feature, describe the feature this PR is introducing and design decisions.
- If it's a refactoring, describe why it is necessary. What are its pros and cons in respect to the previous code, and other possible design choices.

## PR review check list

Quality check list that cannot be automatically verified.

- Safety
    - [ ] The changes respect the necessary conditions for safety (https://raiden-network-specification.readthedocs.io/en/latest/smart_contracts.html#protocol-values-constraints)
-  Code quality
    - [ ] Error conditions are handled
    - [ ] Exceptions are propagated to the correct parent greenlet
    - [ ] Exceptions are correctly classified as recoverable or unrecoverable
- Compatibility
    - [ ] State changes are forward compatible
    - [ ] Transport messages are backwards and forward compatible
- Commits
    - [ ] Have good messages
    - [ ] Squashed unecessary commits
- If it's a bug fix:
    - Regression test for the bug
        - [ ] Properly covers the bug
        - [ ] If an integration test is used, it could not be written as a unit test
- Is it a user facing feature/bug fix?
    - [ ] Changelog entry has been added

### CODE REVIEW

File: raiden/settings.py [hunk 1]
```diff
@@ -55,7 +55,7 @@
 
 MIN_REI_THRESHOLD = 100
 
-MONITORING_REWARD = TokenAmount(1)
+MONITORING_REWARD = TokenAmount(5_000_000_000_000_000_000)  # about 1$
```
Comment (rakanalh) [inline raiden/settings.py]: This depends on the price of RDN no?

Comment (palango) [reply raiden/settings.py]: Sure, the question is if we need an oracle, or if it's enough when we update this with every release.

Comment (LefterisJP) [reply raiden/settings.py]: We would definitely need an Oracle. Price fluctuations happen so often that even daily releases would not help here.

Comment (Dominik1999) [reply raiden/settings.py]: I would not go for an oracle before we publish the RC. That sounds like a lot of legal and testing effort. However, the monitoring service is not economical viable anyway at the moment and we will not change that by exactly providing 1$ for every successful defeat or not. So we should go with the easiest solution and change it iteratively after the release.

**Pull request merged**